dbms/src/Storages/Kafka/StorageKafka.cpp

```diff
@@ -410,14 +412,12 @@ bool StorageKafka::streamToViews()
 }
 
 
-bool StorageKafka::hasSetting(const String & setting_name) const
+void StorageKafka::checkSetting(const String & setting_name) const
```

Comment:
Look good to me.
Just one note about this name: `checkSettingCanBeChanged()` seems to be more appropriate.